docs/src/developing/clients/javascript-api.md

```diff
@@ -2,4 +2,308 @@
 title: Web3 JavaScript API
 ---
 
-See [solana-web3](https://solana-labs.github.io/solana-web3.js/).
+## What is Solana-Web3.js?
+
+The Solana-Web3.js library aims to provide complete coverage of Solana. The library was built on top of the [Solana JSON RPC API](https://docs.solana.com/developing/clients/jsonrpc-api).
+
+## Common Terminology
+
+| Term | Definition |
+|-------------|------------------------|
+| Program     | Stateless executeable code written to interpret instructions. Programs are capable of performing actions in based on the instructions provided. |
```

Comment:
> Programs are capable of performing actions in based on the instructions provided

grammar^

Reply:
Updated to `Stateless executeable code written to interpret instructions. Programs are capable of performing actions based on the instructions provided.`